cloudweatherreport/templates/base.html

```diff
@@ -7,7 +7,7 @@
         <link rel="stylesheet" type="text/css" media="screen" href="https://assets.ubuntu.com/v1/vanilla-framework-version-0.0.65.min.css">
         <!-- link rel="stylesheet" type="text/css" media="all"
               href="http://assets.ubuntu.com/sites/guidelines/css/latest/ubuntu-styles.css"/ -->
-        <link rel="stylesheet" type="text/css" href="http://status.juju.solutions/static/css/base.css" >
+        <link rel="stylesheet" type="text/css" href="http://data.vapour.ws/assets/cwr/css/base.css" >
```

Comment:
Why the URL change here?  Also, we should probably just push the CSS to the datastore and use a relative link.